### PR TITLE
Tw 219 update python links

### PR DESCRIPTION
**Pull request by DJSanti**
## What was changed
Legacy links and a few comments linked to long-closed issues were updated or removed.

## Why?
Recent docs changes make it necessary to check other repos for broken links.

### CODE REVIEW

File: custom_converter/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 This sample shows how to make a custom payload converter for a type not natively supported by Temporal.
 
-To run, first see [README.md](../README#usage) for prerequisites. Then, run the following from this directory to start the
+To run, first see [README.md](../README) for prerequisites. Then, run the following from this directory to start the
```
Comment (cretz) [inline custom_converter/README.md:5]: Why was this changed? We want to link to that section.

File: custom_converter/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 This sample shows how to make a custom payload converter for a type not natively supported by Temporal.
 
-To run, first see [README.md](../README.md) for prerequisites. Then, run the following from this directory to start the
+To run, first see [README.md](../README) for prerequisites. Then, run the following from this directory to start the
```
Comment (cretz) [inline custom_converter/README.md:5]: This breaks GitHub. For example, click the link in your branch: https://github.com/temporalio/samples-python/tree/TW-219-update-python-links/custom_converter

**Comment by DJSanti** [comment]
> CI is failing because the Python format doesn't match since the changes. Can run `poe format` to get files back formatted properly.

How do I configure that? I can't get it working on my end.

**Comment by cretz** [comment]
Follow the test instructions at the root of the repo for regular install (so `poetry install`) then follow the instructions for the test part which tell you how to install `poe`. Then you should be able to run `poe format`.

File: custom_converter/worker.py [hunk 3]
```diff
@@ -65,7 +65,6 @@ def __init__(self) -> None:
         # Just add ours as first before the defaults
         super().__init__(
             GreetingEncodingPayloadConverter(),
-            # TODO(cretz): Make this list available without instantiation - https://github.com/temporalio/sdk-python/issues/139
```
[the remark this answers is not in the document]
Comment (cretz) [reply custom_converter/worker.py:68]: Marking unresolved, I think this TODO needs to stay in the source until we come back and fix it

File: hello/hello_activity_multiprocess.py [hunk 4]
```diff
@@ -9,7 +9,6 @@
 from temporalio.client import Client
 from temporalio.worker import SharedStateManager, Worker
 
-# TODO(cretz): https://github.com/temporalio/sdk-python/issues/201
```
[the remark this answers is not in the document]
Comment (cretz) [reply hello/hello_activity_multiprocess.py:12]: Marking unresolved, I think this TODO needs to stay in the source until we come back and fix it

File: hello/hello_activity_threaded.py [hunk 5]
```diff
@@ -8,7 +8,6 @@
 from temporalio.client import Client
 from temporalio.worker import Worker
 
-# TODO(cretz): https://github.com/temporalio/sdk-python/issues/201
```
[the remark this answers is not in the document]
Comment (cretz) [reply hello/hello_activity_threaded.py:11]: Marking unresolved, I think this TODO needs to stay in the source until we come back and fix it

**Comment by DJSanti** [comment]
Okay, looks like the comments weren't back until just now. Should be good to go @cretz @flossypurse .

File: custom_decorator/README.md [hunk 6]
```diff
@@ -3,7 +3,7 @@
 This sample shows a custom decorator can help with Temporal code reuse. Specifically, this makes a `@auto_heartbeater`
 decorator that automatically configures an activity to heartbeat twice as frequently as the heartbeat timeout is set to.
 
-To run, first see [README.md](../README.md) for prerequisites. Then, run the following from this directory to start the
+To run, first see [README.md](../README) for prerequisites. Then, run the following from this directory to start the
```
[the remark this answers is not in the document]
Comment (DJSanti) [reply custom_decorator/README.md:6]: No idea why this isn't reflected in my editor. Going to check this out.

File: custom_converter/README.md (hunk 2 above)
Comment (cretz) [reply custom_converter/README.md:5]: The extension is still removed here

**Review by flossypurse — APPROVED** [review]

**Comment by flossypurse** [comment]
lgtm - wait for approval from @cretz though

**Review by cretz — APPROVED** [review]
Looks great! Merging.

**Pull request merged**